components/homme/src/share/cxx/GllFvRemapImpl.cpp

```diff
@@ -395,7 +395,7 @@ ::run_dyn_to_fv_phys (const int timeidx, const Phys1T& ps, const Phys1T& phis, c
   const auto hvcoord = m_hvcoord;
   
   const bool use_moisture = m_data.use_moisture;
-  const bool theta_hydrostatic_mode = m_data.theta_hydrostatic_mode;
+  const int theta_hydrostatic_mode = m_data.theta_hydrostatic_mode;
```

Comment:
I believe this is the only place where you need this to be an int, right? I see it used inside the KOKKOS_LAMBDA below. Then, you can keep `theta_hydrostatic_mode` as a bool _everywhere_, and just use an int for the local var here.